### DISCUSSION TITLE
Popup about copying the link

### DISCUSSION THREAD

**Discussion by sazovsky** [roam-garden/i#82] [Feature requests & Ideas]
When I click on the link, I don't know if the link was copied to my clipboard.
![image](https://user-images.githubusercontent.com/86047632/132457101-c8d7ec1c-7c38-404c-9a9a-af4f639bef96.png)

**Comment by Stvad** [comment]
Thanks, I've though of doing that but initially settled for a more subtle feedback (button changing color on click). I'll consider making it more explicit!
